prov/efa/src/rxr/rxr_pkt_cmd.c

```diff
@@ -361,6 +364,86 @@ ssize_t rxr_pkt_post_ctrl_or_queue(struct rxr_ep *ep, int entry_type, void *x_en
 	return err;
 }
 
+/*
+ * This function is used for any extra feature that does not have an alternative,
+ * for example FI_DELIVERY_COMPLETE
+ *
+ * This function will send a eager rtw packet with no data. Then keep calling
```

Comment:
Can you expand on this first sentence to justify the use of `RXR_EAGER_RTW_PKT` over other packet types? In theory, this function could have used an eager RTM packet too, but calling out what RTW gets you here would be more useful than the rest of the comment (which is repeating what the code does).